source/base/ByteBuffer.ooc

```diff
@@ -15,21 +16,24 @@ ByteBuffer: class {
 	_size: Int
 	_referenceCount: ReferenceCounter
 	_ownsMemory: Bool
+	_allocator: AbstractAllocator = null
 	pointer ::= this _pointer
 	size ::= this _size
 	referenceCount ::= this _referenceCount
 
-	init: func (=_pointer, =_size, ownsMemory := false) {
+	init: func (=_pointer, =_size, ownsMemory := false, allocator := null as AbstractAllocator) {
```

Comment:
For some reason `allocator: AbstractAllocator = null` does not compile here, it is somehow affected by the previous `false` default argument:
> base/ByteBuffer.ooc:24:47 error Couldn't unwrap decl `false: AbstractAllocator` , trail = 
\_, Module ByteBuffer
  \_, ClassDecl ByteBuffer 
    \_, ClassDecl ByteBufferClass 
      \_, static ByteBuffer new(=_pointer, =_size, ownsMemory: <unknown type> = <CommaSequence>(false: AbstractAllocator, allocator: AbstractAllocator = null)) -> ByteBuffer
        \_, ownsMemory: <unknown type> = <CommaSequence>(false: AbstractAllocator, allocator: AbstractAllocator = null)
          \_, <CommaSequence>(false: AbstractAllocator, allocator: AbstractAllocator = null)